app/main.py

```diff
@@ -1,2 +1,54 @@
+from copy import copy
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
+        self.hash_table = [[]] * self.capacity
```

Comment:
You can't do that with list of mutable objects: it will produce a list of lists with the same reference.
```python
>>> x = [[]] * 2
>>> x[0] is x[1]
True
```
Your solution still works because you reassign those list items later, but it's bad practice anyway. Use `[[] for _ in range()]` instead.

Reply:
I agree